spec/validators/work_subtype_validator_spec.rb

```diff
@@ -54,7 +54,7 @@
     end
 
     context 'with two valid subtypes and one or more valid "more" types' do
-      let(:value) { %w[Data Sound Preprint] }
+      let(:value) { %w[Data Speech Preprint] }
 
```

Comment:
`Sound` is no longer a valid subtype for mixed media, just replace it with a different one that is valid